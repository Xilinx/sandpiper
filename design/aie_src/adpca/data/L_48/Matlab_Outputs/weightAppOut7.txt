-4.714097977e+00 -1.472896099e+00
-1.239450455e+00 -1.767592907e+00
9.085831642e-01 -1.211563826e+00
