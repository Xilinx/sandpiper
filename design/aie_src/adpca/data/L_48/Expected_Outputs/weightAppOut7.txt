-4.713989258e+00 -1.472867966e+00 
-1.239375830e+00 -1.767399549e+00 
9.085949659e-01 -1.211528063e+00 
